frontend/src/metabase/api/entity-id.ts

```diff
@@ -5,7 +5,7 @@ import {
 
 import { Api } from "./api";
 
-const validEntityTypes = [
+export const VALID_ENTITY_TYPES = [
```

Comment:
Do you use this variable outside this file, otherwise, we shouldn't export it.

```suggestion
const VALID_ENTITY_TYPES = [
```

Reply:
I did at some point. I'll revert this change completely